ome_zarr/scale.py

```diff
@@ -19,10 +20,14 @@
     resize,
 )
 
+from .dask_utils import resize as dask_resize
 from .io import parse_url
 
 LOGGER = logging.getLogger("ome_zarr.scale")
 
+ListOfArrayLike = Union[List[da.Array], List[np.ndarray]]
+ArrayLike = Union[da.Array, np.ndarray]
```

Comment:
Perhaps in the future these could be [some kind of general Array]( https://data-apis.org/array-api/latest/design_topics/static_typing.html ) (once that is more fleshed out).